single_tech_samples/storage/sample1_data_lifecycle/README.md

```diff
@@ -0,0 +1,224 @@
+
+### 1.3. Technologies used
+
+The following technologies are used to build this sample:
+
+- [Azure Storage](https://azure.microsoft.com/services/storage/data-lake-storage)
+- [Terraform](https://www.terraform.io)
+
+## 2. How to use this sample
+
+This section holds the information about usage instructions of this sample.
+
+### 2.1. Prerequisites
+
+The following are the prerequisites for deploying this sample:
+
+- [Github account](https://github.com)
+- [Azure Account](https://azure.microsoft.com/free/search/?&ef_id=Cj0KCQiAr8bwBRD4ARIsAHa4YyLdFKh7JC0jhbxhwPeNa8tmnhXciOHcYsgPfNB7DEFFGpNLTjdTPbwaAh8bEALw_wcB:G:s&OCID=AID2000051_SEM_O2ShDlJP&MarinID=O2ShDlJP_332092752199_azure%20account_e_c__63148277493_aud-390212648371:kwd-295861291340&lnkd=Google_Azure_Brand&dclid=CKjVuKOP7uYCFVapaAoddSkKcA)
+  - *Permissions needed*:  The ability to create and deploy to an Azure [resource group](https://docs.microsoft.com/azure/azure-resource-manager/management/overview), a [service principal](https://docs.microsoft.com/azure/active-directory/develop/app-objects-and-service-principals), and grant the [collaborator role](https://docs.microsoft.com/azure/role-based-access-control/overview) to the service principal over the resource group.
+
+  - Active subscription with the following [resource providers](https://docs.microsoft.com/azure/azure-resource-manager/management/azure-services-resource-providers) enabled:
+    - Microsoft.Storage
+
+#### 2.1.1 Software Prerequisites
+
+- [Azure CLI](https://docs.microsoft.com/cli/azure) installed on the local machine
+  - *Installation instructions* can be found [here](https://docs.microsoft.com/cli/azure/install-azure-cli)
+- [Terraform](https://www.terraform.io) installed on the local machine
+  - *Installation instructions* can be found [here](https://developer.hashicorp.com/terraform/downloads)
+  - Minimum required version is 1.3.x or later
+
+### 2.2. Setup and deployment
+
+> **IMPORTANT NOTE:** As with all Azure Deployments, this will **incur associated costs**. Remember to teardown all related resources after use to avoid unnecessary costs. See [here](#4.3.-deployed-resources) for a list of deployed resources.
+
+Below listed are the steps to deploy this sample :
+
+- Fork and clone this repository. Navigate to  `single_tech_samples/storage/sample1_data_lifecycle/terraform`.
+
+- The sample depends on the variables to be set before the deployment script is run; You can set these variables by editing the variables.tf and terraform.auto.tfvars files, otherwise the default values provided would be considered.
+
+The script is configuration driven and based on a variable of type map(map(map))).
+
+- First level map represents storage accounts.
+- Second level map represents the containers inside the respective storage account.
+- Third level map represents the data life cycle rules under respective container.
+
+Sample variable format is shown below:
+
+```json
```

Comment:
Just mark it as "txt" instead of "json" to avoid weird colouring.

Reply:
Done